cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/security/spnego/SpnegoUserStoreAuthorizationService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.security.spnego;
+
+import com.linkedin.kafka.cruisecontrol.servlet.security.UserStoreAuthorizationService;
+import org.eclipse.jetty.security.UserStore;
+import org.eclipse.jetty.server.UserIdentity;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class SpnegoUserStoreAuthorizationService extends UserStoreAuthorizationService {
+
+  public SpnegoUserStoreAuthorizationService(String privilegesFilePath) {
+    super(privilegesFilePath);
+  }
+
+  public SpnegoUserStoreAuthorizationService(UserStore userStore) {
+    super(userStore);
+  }
+
+  @Override
+  public UserIdentity getUserIdentity(HttpServletRequest request, String name) {
```

Comment:
Well, actually the superclass shouldn't interpret the name as a kerberos one, so in fact I should change the implementation there and this one should stay. I think it could have been a copy-paste mistake.